packages/p2p/src/components/my-profile/my-profile-content.jsx

```diff
@@ -7,6 +7,7 @@ import { useStores } from 'Stores';
 import MyProfileForm from './my-profile-form';
 import MyProfileStats from './my-profile-stats';
 import PaymentMethods from './payment-methods';
+import BlockUserEmpty from '../advertiser-page/block-user/block-user-empty';
```

Comment:
`nit`: May be we can use absolute path relative to the project rather than relative import? Like `import BlockUserEmpty from 'Components/advertiser-page/block-user/block-user-empty';`

Reply:
Got it! Refactored changes